Packs/FeedURLhaus/Integrations/FeedURLhaus/README.md

```diff
@@ -0,0 +1,7 @@
+## URLhaus
```

Comment:
for the reademe, revert to the previous version of the file, you have in the first pr.

Reply:
I dont understand it is the same as it was in the last pr, do I need to add something ? @merit-maita